docs/installation/native-installation.md

```diff
@@ -45,15 +45,28 @@ php phing build-demo-dev
 
 ## Run integrated HTTP server
 
-```sh
-php bin/console server:run
+First follow [installation instructions](https://symfony.com/doc/current/setup/symfony_server.html#installation) for the `symfony` binary which provides its own web server.
+
+Then you can run integrated webserver for the first domain:
+```bash
+symfony server:start
```

Comment:
You have updated phing target server-run for that would not be better to suggest that?

Reply:
This is a good point, but I'm afraid of the confusion if for the first domain it would be phing target and for the other symfony command (as it's not possible to pass an argument into the phing exec command, afaik).

And in order to run properly second domain, `domain_urls.yml` has to be changed, so the second phing target is up to no good.